packages/flutter_test/lib/src/binding.dart

```diff
@@ -834,8 +834,8 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   @override
   void initInstances() {
     super.initInstances();
-    window.onBeginFrame = null;
-    window.onDrawFrame = null;
+    window.onBeginFrame = (Duration _) {};
+    window.onDrawFrame = () {};
```

Comment:
maybe we should track if we've registered our callbacks separately from whether they're null, to avoid this kind of breakage...
Also, we could override registerFrameCallbacks here to enable the test harness and leave `window` entirely alone, thus avoiding this issue.